samples/boards/nordic/system_off/src/retained.c

```diff
@@ -70,6 +74,10 @@ static int ram_range_retain(const void *ptr,
 			    size_t len,
 			    bool enable)
 {
+#if CONFIG_SOC_NRF54H20_CPUAPP
+	/* Core is using global RAM memory which will be retained. Nothing to be done here. */
```

Comment:
That's not fully correct. The System Controller is supposed to disable the retention of global RAM after all local CPUs enter Soft Off.